src/models/pieChart.js

```diff
@@ -66,8 +66,8 @@ nv.models.pieChart = function() {
             nv.utils.initSVG(container);
 
             var that = this;
-            var availableWidth = nv.utils.availableWidth(width, container, margin),
-                availableHeight = nv.utils.availableHeight(height, container, margin);
+            var availableWidth = nv.utils.availableWidth(pie.width(), container, margin),
+                availableHeight = nv.utils.availableHeight(pie.height(), container, margin);
```

Comment:
@liquidpele I still have the sizing issue with that fix. Why it's a better fix than the other? I saw that every charts have width and height added in there option, pie chart is the only one missing...